api/src/hyp3_api/openapi-spec.yml

```diff
@@ -37,6 +37,22 @@ components:
         - jobId
         - parameters
       additionalProperties: false
+    submission:
+      type: object
+      properties:
+        process_type:
+          type: string
+          example: RTC_GAMMA
+        email:
+          type: string
+          example: john.doe@example.com
```

Comment:
considered a pattern, not sure if worth it as emails can actually be pretty tricky to match well

Reply:
`type: email` is a build-in jsonschema type
https://json-schema.org/understanding-json-schema/reference/string.html#email-addresses

Reply:
Makes our test fail... Sad day

Reply:
https://swagger.io/specification/#data-type-format

looks like this might work instead:
```
email:
  type: string
  format: email
  example: john.doe@example.com
```